src/DynamoCoreWpf/ViewModels/Preview/WatchViewModel.cs

```diff
@@ -272,7 +272,15 @@ private static string GetStringFromObject(object obj)
                 case TypeCode.DateTime:
                     return ((DateTime)obj).ToString(PreferenceSettings.DefaultDateFormat, CultureInfo.InvariantCulture);
                 case TypeCode.Object:
+                    if (obj is byte[] byteArray)
```

Comment:
can you add tests for the cases you are fixing?

Reply:
Yes, I was going to add tests, which is why I didn't call this out for review yet but thanks for the review all the same.